.eslintrc.js

```diff
@@ -1,4 +1,10 @@
 module.exports = {
   root: true,
   extends: '@react-native-community',
```

Comment:
it looks like I broke the whole prettier, not just the rule.  I'll fix that later.